rancher/distros-test-framework
# fix: docker build var and adding PR template

#### Proposed Changes ####

- fix https://github.com/rancher/distros-test-framework/issues 
- adding PR template




#### Types of Changes ####



#### Verification ####



#### Testing ####




Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
- yes 

2. Verify code lint; we should not have errors.

- ok 
3. Update the documentation if needed.

- ok 

4. Update makefile and docker run if adding new tests.

- ok

5. Run your tests at least 4 times with all configurations needed and possible.

-n/a
#### Linked Issues ####



#### User-Facing Change ####

```release-note

```

#### Further Comments ####
